dora/tests/integration/src/test/java/alluxio/client/cli/fsadmin/BackupCommandStateLockingIntegrationTest.java

```diff
@@ -1,105 +0,0 @@
-/*
- * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
- * (the "License"). You may not use this work except in compliance with the License, which is
- * available at www.apache.org/licenses/LICENSE-2.0
- *
- * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied, as more fully set forth in the License.
- *
- * See the NOTICE file distributed with this work for information regarding copyright ownership.
- */
-
-package alluxio.client.cli.fsadmin;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotEquals;
-import static org.junit.Assert.assertTrue;
-
-import alluxio.AlluxioURI;
-import alluxio.annotation.dora.DoraTestTodoItem;
-import alluxio.conf.Configuration;
-import alluxio.conf.PropertyKey;
-import alluxio.exception.AlluxioException;
-import alluxio.exception.ExceptionMessage;
-import alluxio.master.MasterContext;
-import alluxio.master.MasterProcess;
-import alluxio.master.StateLockManager;
-import alluxio.master.StateLockOptions;
-import alluxio.resource.LockResource;
-import alluxio.testutils.LocalAlluxioClusterResource;
-
-import org.junit.Ignore;
-import org.junit.Test;
-import org.powermock.reflect.Whitebox;
-
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-
-/**
- * Integration tests for the backup command's interaction with the state-locking.
- */
-@LocalAlluxioClusterResource.ServerConfig(confParams = {
-    PropertyKey.Name.MASTER_BACKUP_DIRECTORY, "${alluxio.work.dir}/backups",
-    PropertyKey.Name.MASTER_SHELL_BACKUP_STATE_LOCK_TRY_DURATION, "3s",
-    PropertyKey.Name.MASTER_SHELL_BACKUP_STATE_LOCK_TIMEOUT, "3s",
-    PropertyKey.Name.MASTER_SHELL_BACKUP_STATE_LOCK_GRACE_MODE, "TIMEOUT"})
-@Ignore
-@DoraTestTodoItem(action = DoraTestTodoItem.Action.REMOVE, owner = "jiacheng",
```

Comment:
the backup feature no longer matters in 3.x. The master will no longer have a journal. The journal code still exits as of now but will be removed soon.